doc/rant/integer-types.md

```diff
@@ -52,7 +52,7 @@ So I then became a convert to the idea that devs should use `stdint.h /
 inttypes.h` for everything.
 
 But I then learned the hard way it is a fool's errand to try to convince
-other developers to religiously to do this (nobody likes typing `printf(
+other developers to religiously do this (nobody likes typing `printf(
```

Comment:
Hmm your change introduces a split infinitive. I think "to do this religiously" sounds the best.